packages/components/src/components/Passphrase/PassphraseTypeCard.tsx

```diff
@@ -277,157 +264,156 @@ export const PassphraseTypeCard = (props: PassphraseTypeCardProps) => {
             }}
             data-test={`@passphrase-type/${props.type}`}
         >
-            {!props.singleColModal && (
-                // only used to show options in modal where user selects wallet type
-                // single col modal such as one for creating hidden wallet shows only input and submit button
-                <>
-                    <Row>
-                        <IconWrapper $type={props.type}>
-                            {props.type === 'standard' ? (
-                                <Icon size={24} icon="WALLET" color={theme.iconPrimaryDefault} />
-                            ) : (
-                                <Icon size={24} icon="LOCK" color={theme.iconSubdued} />
+            <Item>
```

Comment:
This component is quite long, can we split it into some smaller, well named components?